tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -128,7 +128,7 @@ Create VCH - defaults
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     ${output}=  Run  bin/vic-machine-linux inspect config --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT}
-    Should Contain  ${output}  --debug=1
+    Should Contain  ${output}  --debug=-1
```

Comment:
with ^^^ comments, you'll need to remove this check. atm, it will have to check VCH logs to see if DEBUG is enabled.
atm, it might be okay without check in CI I guess.